engine/src/flutter/shell/platform/darwin/macos/framework/Source/FlutterDisplayLinkTest.mm

```diff
@@ -33,38 +34,45 @@ - (void)onDisplayLink:(CFTimeInterval)timestamp targetTimestamp:(CFTimeInterval)
 
 @end
 
-TEST(FlutterDisplayLinkTest, ViewAddedToWindowFirst) {
+class FlutterDisplayLinkTest : public testing::Test {
+ public:
+  void SetUp() override { [FlutterRunLoop ensureMainLoopInitialized]; }
+};
+
+TEST_F(FlutterDisplayLinkTest, ViewAddedToWindowFirst) {
   NSWindow* window = [[NSWindow alloc] initWithContentRect:NSMakeRect(0, 0, 100, 100)
                                                  styleMask:NSWindowStyleMaskTitled
                                                    backing:NSBackingStoreNonretained
                                                      defer:NO];
   NSView* view = [[NSView alloc] initWithFrame:NSMakeRect(0, 0, 100, 100)];
   [window setContentView:view];
 
-  auto event = std::make_shared<fml::AutoResetWaitableEvent>();
+  __block bool signalled = false;
```

Comment:
nit, here and below: Unless we're passing to C++ calls that take a `bool`, prefer `BOOL` / `YES` (below).